nav2_msgs/msg/CostmapFilterSemanticInfo.msg

```diff
@@ -0,0 +1,10 @@
+std_msgs/Header header
+# Type of plugin used (keepout filter, speed limit in m/s, speed limit in percent, etc...)
+uint8 type
+# Name of map filter topic
+string map_filter_topic
+# Multiplier base offset and multiplier coefficient for conversion of OccGrid
```

Comment:
Add some comment like: "Used to convert image translated values into filter needed values"

Reply:
Made in https://github.com/ros-planning/navigation2/commit/fcf337566456677d0a6892b8539e902dfd086994#diff-ea61287bc0126908adac952fac129bc2R7